src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/RazorMapToDocumentEditsParams.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using MediatR;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer
+{
+    internal class RazorMapToDocumentEditsParams : IRequest<RazorMapToDocumentEditsResponse>
+    {
+        public RazorLanguageKind Kind { get; set; }
```

Comment:
Are setters necessary? Ditto for `RazorMapToDocumentEditsParams.cs` & `RazorMapToDocumentEditsResponse.cs`

Reply:
I believe so. Or we need a constructor.